Installable Versions dropdown is not filled with data due to MalformedJsonException

Hello,

I have a problem with creating project. I go File > New project > Symfony and then I do not get any versions in the dropdown, even after clicking Reload button.

JSON returned by https://symfony.com/versions.json seems to be valid according to RFC 4627.

PhpStorm 2018.2.2
Build #PS-182.4129.45
Symfony Plugin 0.16.165


com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 25 path $
	at com.google.gson.JsonParser.parse(JsonParser.java:65)
	at com.google.gson.JsonParser.parse(JsonParser.java:45)
	at fr.adrienbrault.idea.symfony2plugin.installer.SymfonyInstallerUtil.getVersions(SymfonyInstallerUtil.java:114)
	at fr.adrienbrault.idea.symfony2plugin.installer.SymfonyInstallerForm.getVersions(SymfonyInstallerForm.java:54)
	at fr.adrienbrault.idea.symfony2plugin.installer.SymfonyInstallerForm.lambda$appendSymfonyVersions$3(SymfonyInstallerForm.java:63)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 25 path $
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1568)
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1409)
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:542)
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:425)
	at com.google.gson.JsonParser.parse(JsonParser.java:60)
	... 10 more

--- a/src/main/java/fr/adrienbrault/idea/symfony2plugin/installer/SymfonyInstallerUtil.java
+++ b/src/main/java/fr/adrienbrault/idea/symfony2plugin/installer/SymfonyInstallerUtil.java
@@ -178,7 +178,7 @@ public class SymfonyInstallerUtil {
 
             // @TODO: PhpStorm9:
             // simple replacement for: com.intellij.util.io.HttpRequests
-            URL url = new URL("http://symfony.com/versions.json");
+            URL url = new URL("https://symfony.com/versions.json");
             URLConnection conn = url.openConnection();
             conn.setRequestProperty("User-Agent", userAgent);
             conn.connect();

